assignments/prerequisites/python-programming-basics.ipynb

```diff
@@ -66,7 +66,7 @@
    "cell_type": "markdown",
```

Comment:
<p>You can remove existing code for string upper, lower, title, replace. And we should find some other examples for the. you don't need to cover all of them. Just one or two could be good.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/386/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>